debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -772,12 +773,26 @@ public static SchemaRefreshMode parse(String value) {
                 return 0;
             });
 
-    public static final Field YB_LOAD_BALANCE_CONNECTIONS = Field.create("yb.load.balance.connections")
+    public static final Field YB_LOAD_BALANCE_CONNECTIONS =
+        Field.create("yb.load.balance.connections")
             .withDisplayName("YB load balance connections")
-            .withType(Type.BOOLEAN)
+            .withType(Type.STRING)
             .withDefault(YB_DEFAULT_LOAD_BALANCE_CONNECTIONS)
             .withImportance(Importance.LOW)
-            .withDescription("Whether or not to add load-balance property to connection url");
+            .withDescription(
+                    "This config determines load-balance property in the connection url. " +
+                    "Supported values are 'true', 'only-primary', 'only-rr', 'prefer-primary', " +
+                    "'prefer-rr' and 'false'")
+            .withValidation((config, field, output) -> {
+                final String value = config.getString(field);
+                Set<String> validValues = Set.of("true", "only-primary", "only-rr", "prefer-primary", "prefer-rr", "false");
+                if (!validValues.contains(value)) {
+                    output.accept(field, value,
+                            "The valid values of yb.load.balance.connections are " + validValues);
+                    return 1;
+                }
+                return 0;
```

Comment:
Add a UT to validate that connector throws an error when bogus value is provided for the config `yb.load.balance.connections`